pom.xml

```diff
@@ -186,7 +200,7 @@
                 <artifactId>scalastyle-maven-plugin</artifactId>
                 <version>1.0.0</version>
                 <configuration>
-                    <verbose>false</verbose>
+                    <verbose>true</verbose>
```

Comment:
Not sure but may want to switch this back to verbose = false before merging the PR?